services/client-app/src/routes/index.tsx

```
@@ -2,8 +2,14 @@ import { createFileRoute } from '@tanstack/react-router'
 import AuthContainer from '@/components/auth/auth-container'
 import { useAuth } from '@/lib/auth-context'
 import AssetsContainer from '@/components/assets/assets-container'
+
 export const Route = createFileRoute('/')({
   component: HomeRoute,
+  validateSearch: (search: { tag?: string | undefined | {} }) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)